tox.ini

```diff
@@ -5,10 +5,10 @@ envlist = {py26,py27,pypy}-{lowest,release,devel}{,-simplejson}, {py33,py34,py35
 
 [testenv]
 commands =
-    py.test []
-
+    py.test --cov=flask --cov-report html {toxinidir}/tests/
```

Comment:
Also the `[]` that you removed allows us to pass parameters `a b c` to py.test using `tox -epy27 -- a b c`.